tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -53,9 +55,11 @@ sub run() {
     type_string 'dummy-password';
     send_key 'alt-y';
     type_string 'dummy-password';
+    wait_still_screen 3;
 
     # OK => Exit
     send_key "alt-o";
+    wait_still_screen 600;
```

Comment:
Here you are checking that screen will not change for 600 seconds (almost the same as "sleep 600"!)